src/modules/menu/components/RecommendedFoodCard.tsx

```diff
@@ -31,46 +31,44 @@ const StyledImage = styled(Image)`
   height: 100%;
   object-fit: cover;
   object-position: center;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  filter: brightness(0.8);
+`;
+
+const AspectRatioSquare = styled.div`
   position: relative;
+  width: 100%;
+  height: 0;
```

Comment:
What does this line do? Does we need it?

Reply:
https://css-tricks.com/aspect-ratio-boxes/

Reply:
it is css trick to get 1/1 aspect ratio 
component css padding-top : x%  this x is percentage of width
so this component wil tall as same as it width

Reply:
Got it, good job